modules/querier/http.go

```diff
@@ -24,22 +24,33 @@ func (q *Querier) TraceByIDHandler(w http.ResponseWriter, r *http.Request) {
 
 	vars := mux.Vars(r)
 	traceID, ok := vars[TraceIDVar]
-
 	if !ok {
 		http.Error(w, "please provide a traceID", http.StatusBadRequest)
 		return
 	}
 
+	isSharded := false
```

Comment:
A few things:
- Open a PR to add these changes separately.
- Include a parameter to indicate whether or not it should search ingesters. 
- Add the basic content negotiation we discussed.